### PR TITLE
Project cleanup

### PR DESCRIPTION
**Pull request by bording**
This cleans up a few things in preparation for my next PR which will have this produce a source package instead, which would remove the need to merge this dependency elsewhere.

### CODE REVIEW

**Comment by Scooletz** [comment]
Ping @Particular/servicecontrol-maintainers Some clean up related to the packing that would be great to have on the board.

**Comment by Scooletz** [comment]
Ping @Particular/servicecontrol-maintainers Could you review and possibly merge it?

File: src/ServiceControl.Monitoring.Data/ServiceControl.Monitoring.Data.csproj [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
   </PropertyGroup>
+
   <PropertyGroup>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <Description>A ServiceControl.Monitoring wire format protocol serializer and a reporter.</Description>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <PackageProjectUrl>https://particular.net/</PackageProjectUrl>
```
Comment (mikeminutillo) [inline src/ServiceControl.Monitoring.Data/ServiceControl.Monitoring.Data.csproj]: Isn't this supposed to include the package name?

Comment (Scooletz) [reply src/ServiceControl.Monitoring.Data/ServiceControl.Monitoring.Data.csproj]: Being given that https://github.com/Particular/ServiceControl.Monitoring.Data/blob/ada4931bac4eee6c5f6f2fc665c24cbab8437a86/src/Directory.Build.targets#L11 already contains a link to the package, I'll remove it and check the output build result.

Comment (Scooletz) [reply src/ServiceControl.Monitoring.Data/ServiceControl.Monitoring.Data.csproj]: This is not a public project and is shipped only as source, internally, to our MyGet. It's not important.

**Comment by Scooletz** [comment]
Just noticed that #9 contains all the changes but targets this branch. If @Particular/servicecontrol-maintainers retarget it to `develop`, this PR (#8) can be closed and you could just merge #9

**Pull request closed without merging**